src/Esfa.Vacancy.Register.Api/Mappings/ApprenticeshipMapper.cs

```diff
@@ -62,6 +62,26 @@ public ApprenticeshipVacancy MapToApprenticeshipVacancy(Domain.Entities.Apprenti
             return apprenticeship;
         }
 
+        private WageUnit MapWageUnit(int? wageUnitId)
+        {
+            if (wageUnitId.HasValue == false)
+            {
+                return WageUnit.Unspecified;
+            }
+
+            switch (wageUnitId.Value)
+            {
+                case 2:
+                    return WageUnit.Weekly;
+                case 3:
+                    return WageUnit.Monthly;
+                case 4:
+                    return WageUnit.Annually;
+                default:
+                    return WageUnit.Unspecified;
```

Comment:
Just in case a new wage unit is added in the future should we add a case for 1 and have default throw an exception instead?